dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataOwnerApi.kt

```diff
@@ -127,7 +127,10 @@ interface DataOwnerApi {
         method = [RequestMethod.HEAD],
         value = ["/{companyId}/data-owners/{userId}"],
     )
-    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PreAuthorize(
```

Comment:
Couldn't this be simplified to just using the "#userId"? 
And then in the DataOwnersManager adding the following function:
un isUserDataOwner(userIdRequester: String): Boolean {
        val userIdAuthenticated = SecurityContextHolder.getContext().authentication.name
        if (userIdAuthenticated == userIdRequester) {
            return true
        } else {
            return false
        }
    }
With is a user would always be able to check ownership for themselves.
What do you think?

Reply:
Is already implemented

Reply:
ok